### DISCUSSION TITLE
How to make custom trained spaCy pipeline more portable?

### DISCUSSION THREAD

**Discussion by funghetto** [explosion/spaCy#11695] [Help: Coding & Implementations] [answered]
I created a pipeline for my needs following the documentation, however the folder it creates is over **600mb** and when I distribute the code to run it it requires the inclusion of **spacy-transformers** as a runtime dependency. 
This is not the case with the official models that are like **20mb** and don't require that heavy dependency, what steps do I need to take to make my pipeline as light as the official ones?

**Comment by polm** [comment, marked as answer]
Some of the pretrained pipelines are small because they don't use transformers. The pretrained transformers pipelines, like `en_core_web_trf`, are over 400MB in size - transformers are just big. If you train a model without transformers it should be much smaller, though it depends on your use of word vectors, for example.
